### PR TITLE
Accessibility: v24.2 changes

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: concepts/05 UI Components/FilterBuilder/10 Accessibility/20 Screen Reader Support.md [hunk 1]
```diff
@@ -0,0 +1 @@
+The Chat component supports screen readers and complies to <a href="https://www.w3.org/WAI/standards-guidelines/aria/" target="_blank">WAI-ARIA</a> standards.
```
Comment (Raushen) [inline concepts/05 UI Components/FilterBuilder/10 Accessibility/20 Screen Reader Support.md]: `The Chat` in the FilterBuilder topic

**Review by pomahtri — APPROVED** [review]

**Review by pomahtri — APPROVED** [review]
Approve for Scheduler

File: concepts/05 UI Components/Chat/10 Accessibility/15 Keyboard Navigation.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+A user can use the following keys to interact with the Chat component: 
+
+<table class="dx-table">
+    <tr>
+        <th>Key</th>
+        <th>Action</th>
+    </tr>
+    <tr>
+        <td>Enter</td>
+        <td>Enters a message.</td>
```
Comment (marker-dao) [inline concepts/05 UI Components/Chat/10 Accessibility/15 Keyboard Navigation.md:10]: The user can use Tab to move between TextArea and SendButton (SendButton is disabled when TextArea does not contain a value, in which case SendButton is not available for focus).

**Review by marker-dao — APPROVED** [review]

File: concepts/05 UI Components/Chat/10 Accessibility/05 Accessibility Features Overview.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
+[note] The overall accessibility level of your application depends on the Chat features that you use.
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/Chat/10 Accessibility/05 Accessibility Features Overview.md]:
```suggestion
[note] The overall accessibility level of your application depends on the Chat features you use.
```

File: concepts/05 UI Components/FilterBuilder/10 Accessibility/05 Accessibility Features Overview.md [hunk 4]
```diff
@@ -0,0 +1,55 @@
+[note] The overall accessibility level of your application depends on the FilterBuilder features that you use.
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/FilterBuilder/10 Accessibility/05 Accessibility Features Overview.md]:
```suggestion
[note] The overall accessibility level of your application depends on the FilterBuilder features you use.
```

File: concepts/05 UI Components/Pagination/10 Accessibility/05 Accessibility Features Overview.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
+[note] The overall accessibility level of your application depends on the Pagination features that you use.
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/Pagination/10 Accessibility/05 Accessibility Features Overview.md]:
```suggestion
[note] The overall accessibility level of your application depends on the Pagination features you use.
```

File: concepts/05 UI Components/Scheduler/10 Accessibility/05 Accessibility Features Overview.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
+[note] The overall accessibility level of your application depends on the Scheduler features that you use.
```
Comment (dirk-pieterse) [inline concepts/05 UI Components/Scheduler/10 Accessibility/05 Accessibility Features Overview.md]:
```suggestion
[note] The overall accessibility level of your application depends on the Scheduler features you use.
```

**Review by dirk-pieterse — APPROVED** [review]

**Pull request merged**